cfme/tests/middleware/test_middleware_server.py

```diff
@@ -48,6 +49,29 @@ def test_list_provider_servers(provider):
          .format(ui_servers, db_servers, mgmt_servers))
 
 
+def test_server_details(provider):
+    """Tests server details on UI
+
+    Steps:
+        * Get servers list from UI
+        * Select each server details in UI
+        * Compare selected server UI details with CFME database and MGMT system
+    """
+    server_list = MiddlewareServer.servers(provider=provider)
+    for server in get_random_list(server_list, 1):
+        srv_ui = server.server(method='ui')
+        srv_db = server.server(method='db')
+        srv_mgmt = srv_ui.server(method='mgmt')
```

Comment:
You _can_ do this with soft asserts, which would allow you to run all the assertions, even if you get one failure, but I will merge this as I don't think it's a necessity for this to be included.